app/src/main/java/com/sthoray/allright/data/model/search/SearchRequest.kt

```diff
@@ -16,6 +16,10 @@ import com.sthoray.allright.utils.SortOrder
  * @property fastShipping 1 if fast shipping is required, else 0
  * @property freeShipping 1 if free shipping is required, else 0
  * @property location the location to search around
+ * @property maxPrice the maximum price of the listings
+ * Client side casts max price to an int, but the server side API accepts float values
```

Comment:
This generates the following output: 

```
maxPrice - the maximum price of the listings Client side casts max price to an int, but the server side API accepts float values
```

See if multiline comments can be done or add some punctuation

Reply:
With HTML tags?